### PR TITLE
Add toDms, toDmm and zerofill methods

### PR DESCRIPTION
**Pull request by dnlkoch**
This adds the methods `dToDms`, `dToDmm` and `zerofill` to the `ProjectionUtil`.

### CODE REVIEW

File: src/ProjectionUtil/ProjectionUtil.js [hunk 1]
```diff
@@ -98,6 +98,49 @@ export class ProjectionUtil {
...
+   *
+   * @param {number} value Value to be converted.
+   *
+   * @return {string} Converted value.
+   */
+  static dToDms(value) {
+    const deg = parseInt(value, 10);
+    const min = parseInt((value - deg) * 60, 10);
+    const sec = ((value - deg - min / 60) * 3600);
+    return `${deg}° ${ProjectionUtil.zerofill(min)}' ${ProjectionUtil.zerofill(sec.toFixed(2))}''`;
```
Comment (marcjansen) [inline src/ProjectionUtil/ProjectionUtil.js]: Shouldn't this end with `"` instead of two single `'`?

File: src/ProjectionUtil/ProjectionUtil.js [hunk 2]
```diff
@@ -98,6 +98,49 @@ export class ProjectionUtil {
...
+  /**
+   * Adds leading zero to all values less than 10 and returns this new
+   * zerofilled value as String. Values which are greater than 10 are not
+   * affected.
+   *
+   * @param {number} value Value to be zerofilled.
+   *
+   * @return {string} converted value with leading zero if necessary.
+   */
+  static zerofill(value) {
```
Comment (marcjansen) [inline src/ProjectionUtil/ProjectionUtil.js]: This doesn't really belong here, does it? Can we have this be defined as const above the class and not be exported?

File: src/ProjectionUtil/ProjectionUtil.js (hunk 1 above)
Comment (dnlkoch) [reply src/ProjectionUtil/ProjectionUtil.js]: It's fine, see [here](https://en.wikipedia.org/wiki/Degree_(angle)#Subdivisions).

File: src/ProjectionUtil/ProjectionUtil.js (hunk 2 above)
Comment (dnlkoch) [reply src/ProjectionUtil/ProjectionUtil.js]: I would leave here for now.

**Pull request merged**